sdcm/cluster_aws.py

```diff
@@ -351,6 +351,11 @@ def network_config_ipv6_workaround_script():  # pylint: disable=invalid-name
             IPv6_CIDR=`curl -s ${BASE_EC2_NETWORK_URL}${MAC}/subnet-ipv6-cidr-blocks`
 
             grep -qi "amazon linux" /etc/os-release || sudo ip route add $IPv6_CIDR dev eth0
+
+            echo "
```

Comment:
what will happen if, for some reason, `IPV6_AUTOCONF="yes"` will already be in place?
will the network be able to work?
isn't better to:
```
if grep -q IPV6_AUTOCONF /etc/sysconfig/network-scripts/ifcfg-eth0:
    sed -i s/IPV6_AUTOCONF="*"/IPV6_AUTOCONF="yes"
else:
    echo "IPV6_AUTOCONF="yes" >> /etc/sysconfig/network-scripts/ifcfg-eth0
```

Reply:
fixed